front/src/store/actions/index.js

```diff
@@ -0,0 +1,7 @@
+import * as api from '../../services/api/filmsDB'
+
+export const fetchFilmList = (filter = new Date()) => ({
```

Comment:
Are you sure that date is only parameter for filter? How do you think this is going to work with server?